CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [0.2.4] - 2024-04-25
+
+- Support the use of AWS Named Profiles for authentication
```

Comment:
Hi @ErebusBat let's add all your changes to the same version, please add the following notes to the CHANGELOG 

- Support for llama370b_instruct_v1
- Explicitly add base64 gem to dependencies